### PR TITLE
Correct discussion links in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dougbu**
- add a link from README.md to CONTRIBUTING.md

nit: remove extra blank lines

### CODE REVIEW

**Comment by dougbu** [comment]
@danroth27 neither of these documents contain links to community projects. Do we have something like that elsewhere e.g. on https://dotnet.microsoft.com/apps/aspnet or in our docs?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,10 +11,10 @@ Security issues and bugs should be reported privately, via email, to the Microso
 ## Other discussions
 Our team members also monitor several other discussion forums:
 
-* [ASP.NET Core forum](https://forums.asp.net/1255.aspx/1?ASP+NET+5)
-* [StackOverflow](https://stackoverflow.com/) with the [`asp.net-core`](https://stackoverflow.com/questions/tagged/asp.net-core), [`asp.net-core-mvc`](https://stackoverflow.com/questions/tagged/asp.net-core-mvc), or [`entity-framework-core`](https://stackoverflow.com/questions/tagged/entity-framework-core) tags.
-* [JabbR chat room](https://jabbr.net/#/rooms/AspNetCore) for real-time discussions with the community and the people who work on the project
```
Comment (dougbu) [inline CONTRIBUTING.md]: Jabbr link results in a 404 and all of these links were for the Core world.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,10 +11,10 @@ Security issues and bugs should be reported privately, via email, to the Microso
...
-
+* [ASP.NET MVC forum](https://forums.asp.net/1146.aspx/1?MVC)
+* [ASP.NET Web API forum](hhttps://forums.asp.net/1246.aspx/1?Web+API)
```
Comment (Eilon) [inline CONTRIBUTING.md]: hhttps

**Comment by Eilon** [comment]
Please fix this ^^^^^^

**Pull request merged**